client/layout/logged-out.jsx

```diff
@@ -127,6 +129,14 @@ const LayoutLoggedOut = ( {
 
 	let masterbar = null;
 
+	// Open new window to create account page when a logged in action was triggered on the Reader tag embed page and the user is not logged in
+	if ( ! isLoggedIn && loggedInAction ) {
+		if ( isReaderTagEmbed ) {
```

Comment:
we could probably combine these if statements

Reply:
Yep, good spot.